tests/testsuite/workspaces.rs

```diff
@@ -2835,8 +2840,8 @@ Available bin targets:
         .with_status(101)
         .with_stderr_data(str![[r#"
 [ERROR] no bin target named `crate2`.
-Available bin targets:
-    another
+Available bin targets in crate2:
```

Comment:
Should we say

> no bin target named `crate2` in crate1 package.

or

> no bin target named `crate2` in default-run packages

Without that extra context, I worry users might get confused because they know might know these targets come from other packages but not realize it

(this should probably be its own commit)